### ISSUE TITLE
Track contrast doesn't meet contrast requirements

### ISSUE THREAD

**Issue by rpearce** [CodyHouse/cd-circular-progress-bar-changelog#1]
The track color of `#e5e5e5` on a `#fff` background has a contrast ratio of `1.25:1`.

<details>
  <summary>Expand to see w3 contrast checker failure info</summary>
<img width="761" alt="W3 contrast checker showing that #e5e5e5 graphical object on a #fff background fails the contrast ratio requirements" src="https://user-images.githubusercontent.com/592876/145071045-c4f885b4-99a8-48ec-9451-aa97c22af12a.png">
</details>

**Comment by claudia-romano** [comment]
Hi there, the contrast we can control is the one between the circular background element (--color-contrast-high with reduced opacity - default is #e3e3e3) and the fill element (--color-contrast-high - default is #1C1C1C) which is 13.27:1. 

According to where this is used (background color and contrast colors you define for your project), you can change the color of the progress bar changing the color of the svg element as explained in the [Info page](https://codyhouse.co/ds/components/info/circular-progress-bar) of the component.

Cheers!
